### DISCUSSION TITLE
Practica: Taqueria

### DISCUSSION THREAD

**Discussion by MauroMontan** [Launch-X-Latam/MisionFrontEnd#264] [Show and tell]
Hola :) 
Les comparto mi taqueria, en este proyecto usé [variables de css](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), sabia de su existencia, pero no de las ventajas que dan, realmente facilitan el trabajo, y es muy cómodo trabajar con ellas. me gustaria saber si conocen algo de css que que sea igual de util !

[Taqueria](https://mauromontan.github.io/taqueria/shop.html)

**Comment by JoseSpelt** [comment]
Buen trabajo broh 👌🏻
